engine/runtime-parser/src/main/java/org/enso/compiler/core/ir/IrPersistance.java

```diff
@@ -389,23 +393,15 @@ public PersistDiagnosticStorage() {
     }
 
     @Override
-    protected void writeObject(DiagnosticStorage obj, Output out) throws IOException {}
+    protected void writeObject(DiagnosticStorage obj, Output out) throws IOException {
+      out.writeInline(List.class, obj.toList());
```

Comment:
Now that the `DiagnosticStorage` is persisted again, how will this affect the overall size of serialized IR? I have a feeling that originally, you said that we do not want to serialize `DiagnosticStorage` as the size of serialized IR would explode.

Reply:
A task for #8323

A quick check shows increase of size from 6618628 to 6685750 for `Standard/Database/Data/Table.ir`.